src/main/java/net/atos/zac/zaaksturing/model/FormulierDefinitie.java

```diff
@@ -14,6 +14,7 @@ public enum FormulierDefinitie {
     AANVULLENDE_INFORMATIE,
     ADVIES(FormulierVeldDefinitie.ADVIES),
     EXTERN_ADVIES_VASTLEGGEN,
+    EXTERN_ADVIES_MAIL,
```

Comment:
Er is een FormulierDefinitie enum en elk formulier heeft public static formulierDefinitie = 'EXTERN_ADVIES_MAIL'; 
De laatste wordt gebruikt in een switch in TaakFormulierenService.getFormulierDefinitie().
Dit voelt dubbel op.
Wanneer jij dat ook zo ziet maak dan een issue aan om dit een keer te refactoren.